.gitignore

```diff
@@ -1,3 +1,6 @@
 gopath/
 bin/
 *.swp
+*.img
```

Comment:
Yes they are, the image files are too large to fit in the tmpfs. Do you have a suggestion for working around that?